include/api.php

```diff
@@ -412,11 +412,13 @@ function pll_count_posts( $lang, $args = array() ) {
 }
 
 /**
- * Allows to access the Polylang instance
- * It is always preferable to use API functions
- * Internal methods may be changed without prior notice
+ * Allows to access the Polylang instance.
+ * However, it is always preferable to use API functions
+ * as internal methods may be changed without prior notice.
  *
  * @since 1.8
+ *
+ * @return PLL_Frontend|PLL_Admin|PLL_Settings|PLL_REST_Request
```

Comment:
This will need to be updated if we ever add a new child class of `PLL_Base`. Alternatively, using `PLL_Base` as the return type will always be true, although a little less specific. Do you believe we'll ever add a new Polylang context / service locator?

Reply:
I have no idea. Initially `PLL_REST_Request` did not exist. `PLL_Base` would not work as we often access methods which exist in only one of the context, so not in `PLL_Base`.